notebooks/ldb-eda-structure.ipynb

```diff
@@ -0,0 +1,1941 @@
+{
```

Comment:
<p>I didn't understand what the code in line 2 does. Besides that, in line 3, you don't need to write <code>test.groupby(test['OverallQual'])</code>. Just <code>test.groupby('OverallQual')</code> already works (it doesn't affect the functionality of the code, but it's more "Pythonic").</p><p> </p><p>What's the target variable scale? I ran your code with the train dataset and the bar plot changed a lot, probably because of that "bug" that happened with SalePrice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>